zipline/data/loader.py

```diff
@@ -182,19 +182,13 @@ def load_market_data(bm_symbol='^GSPC'):
     # we have data,then we need to update
     if len(days_up_to_now) - last_bm_date_offset > 1:
         benchmark_returns = update_benchmarks(bm_symbol, last_bm_date)
-        if (
-            benchmark_returns.index.tz is None
-            or
-            benchmark_returns.index.tz.zone != 'UTC'
-        ):
+        if benchmark_returns.index.tz is None or \
```

Comment:
The PEP8 complaint was that I "or" was at the beginning of a line rather than the end of the previous line.

So if I had changed it as you proposed, it would have looked like this:

    if (benchmark_returns.index.tz is None or
        benchmark_returns.index.tz.zone != 'UTC'):
        ... stuff ...

Unfortunately, PEOP8 would have complained about that because the second line of the `if` conditional is at the same indent level as the first line of the body.

PEP8 is a harsh mistress.